adafruit_azureiot/constants.py

```diff
@@ -0,0 +1,11 @@
+"""This file is for maintaining constants that could be changed or added to over time for different scenarios
+"""
+
+# The version of the IoT Central MQTT API this code is built against
+IOTC_API_VERSION = "2016-11-14"
```

Comment:
`constants` is only imported by `device_registration`, do we need this extra file?

Reply:
The DPS constants are used in `device_registration`, and the IOTC constant is used in `iot_mqtt`. I could split it up and declare the constants in the two separate files but it felt cleaner to have them in one place.

Reply:
It also becomes the one place to check these values such as the endpoint or API version. Makes code navigation easier if this ever needs to change.

Reply:
Agreed, leave it as-is.